examples/model_assisted_labeling/tiled_imagery_mal.ipynb

```diff
@@ -35,34 +35,34 @@
   },
```

Comment:
<p>I think we are just deleting this cell and setting a placeholder. It was causing more trouble than it is worth.</p><p> </p><p><code>API_KEY = None</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>